src/components/AccountDetails/index.tsx

```diff
@@ -265,12 +265,16 @@ export default function AccountDetails({
               <AccountGroupingRow>
                 {formatConnectorName()}
                 <div>
-                  {connector !== injected && connector !== walletlink && (
+                  {/* Coinbase Wallet disconnection causes a page reload from their SDK today. This leads to a race condition where the local storage value for walletOverride may not be set in time. */}
```

Comment:
Does this mean that Coinbase Wallet cannot be disconnected from within the interface page?

Reply:
that's true on prod also, but yeah that would be the case for now